src/vivarium_gates_iv_iron/data/loader.py

```diff
@@ -456,16 +459,66 @@ def load_maternal_disorders_ylds(key: str, location: str) -> pd.DataFrame:
 
 
 def get_hemoglobin_data(key: str, location: str):
-    location_id = utility_data.get_location_id(location) if isinstance(location, str) else location
-    if key == data_keys.HEMOGLOBIN.MEAN:
-        me_id = 10487
-    elif key == data_keys.HEMOGLOBIN.STANDARD_DEVIATION:
-        me_id = 10488
-    else:
-        raise KeyError("Invalid Hemoglobin key")
+    country_dfs = []
+    child_locs = get_child_locs(location)
+
+    for loc in child_locs:
+        location_id = utility_data.get_location_id(loc) if isinstance(loc, str) else loc
+        if key == data_keys.HEMOGLOBIN.MEAN:
+            me_id = 10487
+        elif key == data_keys.HEMOGLOBIN.STANDARD_DEVIATION:
+            me_id = 10488
+        else:
+            raise KeyError("Invalid Hemoglobin key")
+
+        hemoglobin_data = gbd.get_modelable_entity_draws(me_id=me_id, location_id=location_id)
+        hemoglobin_data = reshape_to_vivarium_format(hemoglobin_data, loc)
+        hemoglobin_data = pd.concat([hemoglobin_data], keys=[loc], names=['location'])
+        country_dfs.append(hemoglobin_data)
+
+    national_level_hemoglobin_data = pd.concat(country_dfs)
+
+    return national_level_hemoglobin_data
+
+
+def get_pregnant_lactating_women_location_weights(key: str, location: str):
+    #  WRA * (ASFR + (ASFR * SBR) + incidence_c996 + incidence_c374)
```

Comment:
I know this is a question for Ali, but unless I'm misunderstanding what this quantity represents, I'd expect the numerator to be `WRA * incidence_pregnancy * duration_pregnancy`, while this is just `WRA  * incidence_pregnancy`.

Reply:
This is the calculation of the population for a location, which we divide by the larger regional population to arrive at the location weight. The [research reference](https://vivarium-research.readthedocs.io/en/latest/concept_models/vivarium_iv_iron/maternal_model/concept_model.html#hemoglobin-distribution-weighting-strategy) in the above description describes this.